ui/lib/core/addon/components/overview-card.hbs

```diff
@@ -10,7 +10,7 @@
   data-test-overview-card-container={{@cardTitle}}
   ...attributes
 >
-  <div class="flex row-wrap space-between has-bottom-margin-m" data-test-overview-card={{@cardTitle}}>
```

Comment:
Although this affects overview cards everywhere, it's a very small change and I'd rather the spacing be consistent across the board then have different padding situations.
<img width="1115" alt="Screenshot 2024-07-23 at 1 05 16 PM" src="https://github.com/user-attachments/assets/435639d6-314a-41b6-b3d8-67ed119a2318">